vue3/src/style.css

```diff
@@ -0,0 +1,33 @@
+:root {
+  font-family: 'Noto Sans', Helvetica, Arial, sans-serif;
+  line-height: 1.5;
+  font-weight: 400;
+
+  color-scheme: light dark;
+  color: rgba(255, 255, 255, 0.87);
+  background-color: #242424;
+
+  font-synthesis: none;
+  text-rendering: optimizeLegibility;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+#app {
+  max-width: 1280px;
+  margin: 0 auto;
+  padding: 2rem;
+}
+
+@media (prefers-color-scheme: light) {
```

Comment:
Can you remove this and go with a light theme by default, please? Unfortunately, we currently don't have a dark theme (yet), which means our components won't look great in a dark theme and it will cause a lot of colour contrast accessibility errors.

Reply:
Made some changes -- LMK!